src/components/modals/ManageProjectAddresses/AddNewAddress.tsx

```diff
@@ -39,18 +41,25 @@ export const AddNewAddress: FC<IAddNewAddress> = ({
 		formState: { errors },
 		setError,
 	} = useForm<IAddressForm>({ mode: 'onSubmit', reValidateMode: 'onSubmit' });
+	const chainType =
+		'chainType' in selectedChain ? selectedChain.chainType : undefined;
```

Comment:
How about replacing it with 
`const chainType = selectedChain.chainType` it would have same result

Reply:
The type of selectedChain is "Chain | NonEVMChain" and "chainType" does not exist on type "Chain" so if I use "const chainType = selectedChain.chainType", I get this error: "Property chainType does not exist on type Chain"